pages/lessons/projects/1.mdx

```diff
@@ -297,7 +297,7 @@ contract WAGMI {
     <Callout emoji='💡' size='md' variant='info'>
     While private and internal are not readable or modifiable from other contracts, its values are set on a public blockchain, making its values visible from the outside world.
     </Callout>
- 
+
```

Comment:
It would be awesome if we could remove this change so we don't have to "change" the Lesson in this PR 😊